packages/blocks/src/root-block/widgets/element-toolbar/change-brush-button.ts

```diff
@@ -47,7 +49,7 @@ export class EdgelessChangeBrushButton extends WithDisposable(LitElement) {
   private accessor _selectedColor: string | null = null;
 
   @state()
-  private accessor _selectedSize: LineWidth | null = LineWidth.Four;
+  private accessor _selectedSize: LineWidth | null = null;
```

Comment:
Why not `LineWidth.Four` 🤔

Reply:
The selectedSize getter already has a default value, so no need here